src/Game/Game.h

```diff
@@ -110,10 +117,11 @@ class Game
     bool _quit = false;
     SDL_Event _event;
     bool _initialized = false;
+    std::unique_ptr<InputController> inputController_;
```

Comment:
And one more little remark.... we use **_name** instead of **name_** for private and protected members :) I know that codestyle document isn't ready yet, so it is my bad )